recipes/export2graphlan/meta.yaml

```diff
@@ -0,0 +1,23 @@
+about:
+    home: 'https://bitbucket.org/CibioCM/export2graphlan/overview'
+    summary: "export2graphlan is a conversion software tool for producing both annotation and tree file for GraPhlAn"
+package:
+  name: export2graphlan
+  version: '0.1.0'
```

Comment:
Can you set the version to the one defined [here](https://bitbucket.org/CibioCM/export2graphlan/src/d6c10c11ed4c5a6f2bf930c053b3b12e5473298e/export2graphlan.py?fileviewer=file-view-default)?